### DISCUSSION TITLE
How to get a user theme into standard themes

### DISCUSSION THREAD

**Discussion by LesHol** [israel-dryer/ttkbootstrap#580] [Q&A] [answered]
I have created a function where you can choose which theme to use. I have then created a new theme in ttkcreator and saved it. How do I get it into "standard themes"? I tried to copy the parameters into the standard file but it doesn´t work. 
Error: line 595, in theme_use  raise TclError(themename, "is not a valid theme.")

Please help 
TIA Leslie

**Comment by israel-dryer** [comment, marked as answer]
https://ttkbootstrap.readthedocs.io/en/latest/themes/themecreator/#create-a-new-theme

ttkcreator drops the theme in the USER_THEMES dict (not STANDARD_THEMES).

If you are using the json file, you would use something like the following:

```python
import ttkbootstrap as ttk

style = ttk.Style()
style.load_user_themes("mythemes.json")   # {"themes": [{"mytheme": ....

# or single theme definition
style.load_user_theme(theme_definition)
```

---

For 2.0, this whole area is slated for the semantic-anchor theme-model overhaul, but for 1.x the USER_THEMES dict is the documented standard mechanism.
